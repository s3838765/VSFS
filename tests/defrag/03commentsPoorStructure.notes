NOTES V1.0
=dir1/dir2/
#this is an ignored line
#this is some more ignored line data
@rootfile1
 I am a root file
@dir1/dir2/file1
 This some data
#this is another ignored comment
=dir1/
=dir1/dir3/
@dir1/file2
 The previous file lied!
 I do have data!!!
@dir2/file1
#guess what there is another ignored line
=dir2/
@rootfile2
 I am the brother of the other root file
@dir1/file1
 Some data
 In this file
 The next file has no data
