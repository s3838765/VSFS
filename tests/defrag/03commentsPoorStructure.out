NOTES V1.0
=dir1/
=dir1/dir2/
@dir1/dir2/file1
 This some data
=dir1/dir3/
@dir1/file1
 Some data
 In this file
 The next file has no data
@dir1/file2
 The previous file lied!
 I do have data!!!
=dir2/
@dir2/file1
@rootfile1
 I am a root file
@rootfile2
 I am the brother of the other root file
